README.md

```diff
@@ -64,10 +65,9 @@ func main() {
 
     myLogger.Log("Message 2")
 
-    // Configure options, level and app with specific logs
+    // Configure options with specific logs
```

Comment:
README update:

## Setup
```golang
import (
    "github.com/logdna/logdna-go/logger"
)

func main() {
    key := "YOUR INGESTION KEY HERE"

    // Configure your options with your desired level, hostname, app, ip address, mac address and environment. 
    // Hostname is the only required field in your options- the rest are optional.
    options := logger.Options{}
    options.Level = "fatal"
    options.Hostname = "gotest"
    options.App = "myapp"
    options.IPAddress = "10.0.1.101"
    options.MacAddress = "C0:FF:EE:C0:FF:EE"
    options.Env = "production"
    options.Tags = "logging,golang"

    myLogger, err := logger.CreateLogger(options, key)
}
```
_**Required**_
* [LogDNA Ingestion Key](https://app.logdna.com/manage/profile)

## Usage

After initial setup, logging looks like this:
```golang
func main() {
    ...
    myLogger.Log("Message 1")

    // Can also use Go's short-hand syntax for initializing structs to define all your options in just a single line:
    options = logger.Options{Level: "error", Hostname: "gotest", App: "myapp", IPAddress: "10.0.1.101", MacAddress: "C0:FF:EE:C0:FF:EE"}
    myLogger, err = logger.CreateLogger(options, key)
    myLogger.Log("Message 2")

    // Configure options with specific logs
    newOptions := logger.Options{Level: "warning", Hostname: "gotest", App: "myotherapp", IPAddress: "10.0.1.101", MacAddress:  "C0:FF:EE:C0:FF:EE"}
    errWithOpts := myLogger.LogWithOptions("Message 3", newOptions)

    // We support the following 6 levels
    myLogger.Info("Message 1")
    myLogger.Warn("Message 2")
    myLogger.Debug("Message 3")
    myLogger.Error("Message 4")
    myLogger.Fatal("Message 5")
    myLogger.Critical("Message 6")

    // To add metadata to every log-line created by the logger instance:
    options.Meta = `{"key": "value", "key2": "value2"}`
    myLogger, err = logger.CreateLogger(options, key)
    myLogger.Log("Message 7")
    myLogger.Close()
}
```